src/components/dev-dot-content/mdx-components/mdx-code-blocks/index.tsx

```diff
@@ -86,49 +87,68 @@ function CodeBlockConfig({
  */
 export const MdxCodeBlockConfig = CodeBlockConfig
 
-export function MdxCodeTabs({ children }: { children: ReactNode }) {
+export function MdxCodeTabs({
+	children,
+	heading,
+	tabs,
+}: {
+	children: ReactNode
+	heading?: string
```

Comment:
What's the use case for the `heading` prop here? Only asking cause we didn't previously support this, and wondering if it'd be better for longer-term maintenance if authors could do like:

```markdown
Some text

<CodeTabs ... />
```

rather than

```markdown
<CodeTabs heading="Some text" ... />
```

Reply:
We actually do have docs that are already providing a `heading` prop! [Source](https://github.com/hashicorp/tutorials/blob/384cd5490d4adb7fbeb89aa5e8c5b5cf54324d44/content/tutorials/consul/kubernetes-admin-partitions.mdx?plain=1#L153-L155)

This is what it looked like previously:
![image](https://github.com/hashicorp/dev-portal/assets/88163/48127e2c-d7c6-4dee-8186-1e8dbe108390)

[Link](https://dev-portal-4vkgmt8n5-hashicorp.vercel.app/consul/tutorials/implement-multi-tenancy/kubernetes-admin-partitions#:~:text=These%20files%20are%20configured%20for%20the%20tutorial.%20In%20non%2Dtutorial%20deployments%2C%20make%20note%20of%20the%20highlighted%20lines.%20Your%20organization%20may%20have%20specific%20guidance%20for%20you%20to%20follow%20regarding%20the%20values%20in%20the%20highlighted%20lines.)

Reply:
Ah dang okay I see now you'd also specified that in the testing steps! 🙈 Looks great!